api_tests/nodes/views/test_node_list.py

```diff
@@ -535,6 +536,7 @@ def test_filtering_on_parent(self):
         assert_not_in(parent._id, guids)
         assert_not_in(parent2._id, guids)
 
+    @unittest.skip('Fix filter[parent]=null for NodesList')
```

Comment:
I think in some iteration of working on this I removed support for it... but yes, this test currently passes and shouldn't be skipped. Removing.